### ISSUE TITLE
** Uncaught Exception! **

### ISSUE THREAD

**Issue by KaoGomi** [itdelatrisu/opsu#468]
`**Version:**` 0.16.1
**Build date:** 2018-06-03 19:24
**OS:** Linux (amd64)
**JRE:** 1.8.0_191
**Error:** `** Uncaught Exception! **`
**Stack trace:**
```
java.awt.AWTError: Assistive Technology not found: org.GNOME.Accessibility.AtkWrapper
	at java.awt.Toolkit.loadAssistiveTechnologies(Toolkit.java:807)
	at java.awt.Toolkit.getDefaultToolkit(Toolkit.java:886)
	at org.lwjgl.LinuxSysImplementation.<clinit>(LinuxSysImplementation.java:50)
	at org.lwjgl.Sys.createImplementation(Sys.java:131)
	at org.lwjgl.Sys.<clinit>(Sys.java:116)
	at org.lwjgl.opengl.Display.<clinit>(Display.java:135)
	at org.newdawn.slick.AppGameContainer$1.run(AppGameContainer.java:39)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.newdawn.slick.AppGameContainer.<clinit>(AppGameContainer.java:36)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:195)
```

**Comment by KaoGomi** [comment]
This occurred when I compiled it from the tar.gz with OpenJDK 8 installed. Updated to OpenJDK 11 and I still get an error

**Comment by itdelatrisu** [comment]
Merging into #473.

**Comment by JohnnyLAmpAz** [comment]
> This occurred when I compiled it from the tar.gz with OpenJDK 8 installed. Updated to OpenJDK 11 and I still get an error in a related post in #463.
> 
> Confused as to what I can do now.

So, at first I tried with the latest 0.16.1 `.jar` file while using OpenJDK 11 but i got the same error. Then I cloned the repo and tried to run opsu with `gradle run` which ended up failing at building task with the same error. So I tried to switch to OpenJDK 8 and _tadaa_, it opened! (still through `gradle run`).
After this I thought that maybe the problem was using the 11 version instead with the 8 one, so I tried again to run the release `.jar` with OpenJDK 8 but nothing... I also tried to build the jar with `gradle jar` as the README says but for some reason I cannot run it.

I don't know what this means but I'm writing here this hopefully helpful informations
